src/Main.js

```diff
@@ -0,0 +1,33 @@
+import HornedBeast from './HornedBeast.js';
+
+export default function Main() {
+  return (
+    <main>
+      {beasts.map(function(e) {
+        return <HornedBeast title={e.title} image_url={e.image_url} description={e.description} keyword={e.keyword} />
+      })}
+    </main>
+  );
```

Comment:
Hi Polo, awesome work goin' on here!

- Only feedback I can see is that we could consider using a different name for the callback in the map perhaps, for readability etc.
-  Perhaps something like `beast.map( beast =>`, then all props would have `beast.title, beast.image_url` 
- Just some food for thought here 👍  I know you for sure know what you are doin' here :)